src/ess/bifrost/types.py

```diff
@@ -2,4 +2,15 @@
 
 from scipp import Variable
 
+from ess.reduce.nexus import types as reduce_t
+
 ArcNumber = NewType('ArcNumber', Variable)
+
+# See https://github.com/scipp/essreduce/issues/105 about monitor names
+FrameMonitor0 = reduce_t.Monitor1
+FrameMonitor1 = reduce_t.Monitor2
+FrameMonitor2 = reduce_t.Monitor3
+FrameMonitor3 = reduce_t.Monitor4
+PsdMonitor0 = reduce_t.Monitor5
+PsdMonitor1 = reduce_t.Monitor6
```

Comment:
These two PSD monitors are purely a McStas construction that won't be in the real instrument.
The fourth monitor can be operated in a PSD mode, but likely will only ever be operated in a 0-D mode when collecting inelastic data, so the PSD data may only ever be written to non-standard data files.

Reply:
I think we can skip them entirely for the workflow.